app/static/dayview.css

```diff
@@ -64,46 +84,42 @@ html {
 }
 
 .title_size_tiny {
-
     font-size: 0.2em;
     line-height: 1em;
 }
 
 .actiongrid {
-  grid-row: 1 / -1;
-  grid-column: 1 / -1;
-  display: grid;
-  grid-template-rows: repeat(100, 0.375rem);
-  z-index: 42;
+    grid-row: 1 / -1;
+    grid-column: 1 / -1;
+    display: grid;
+    grid-template-rows: repeat(100, 1fr);
 }
 
 .action-icon {
   visibility: hidden;
 }
 
-.action-continer:hover {
-  border-top: 1px dashed var(--borders);
-  border-bottom: 1px dashed var(--borders);
-  transition: 0.3;
+.action-container {
+    z-index: 70;
 }
 
-.action-continer:hover .action-icon {
-  visibility: visible;
+.event:hover .action-container .action-icon {
+    visibility: visible;
 }
 
 .week-view-title {
     font-size: 2em;
 }
 
 .zodiac-sign {
-  position: fixed;
-  right: 1.2em;
-  top: 1.6em;
-  padding-right: 0.4em;
-  padding-left: 0.4em;
-  padding-bottom: 0.2em;
-  border: solid 0.1px var(--primary);
-  background-color: var(--borders-variant);
-  border-radius: 50px;
-  box-shadow: 1px 1px 2px #999;
+    position: fixed;
+    right: 1.2em;
+    top: 1.6em;
+    padding-right: 0.4em;
```

Comment:
Use vars  if u can.

you can use 1 line to all paddings as such: 
```
padding: top left bottom right; 
```

Reply:
padding-top/padding-right and top/right are not the same thing